actioncable/test/subscription_adapter/redis_test.rb

```diff
@@ -5,6 +5,12 @@ class RedisAdapterTest < ActionCable::TestCase
   include CommonSubscriptionAdapterTest
 
   def cable_config
-    { adapter: 'redis', url: 'redis://127.0.0.1:6379/12' }
+    { adapter: 'redis', driver: 'ruby', url: 'redis://127.0.0.1:6379/12' }
+  end
+end
+
+class RedisAdapterTest::Hiredis < RedisAdapterTest
+  def cable_config
+    super.merge(driver: 'hiredis')
```

Comment:
Does setting `driver` configure faye-websocket somehow? Because I don't understand it 😁